pkg/idp/service_test.go

```diff
@@ -539,7 +544,7 @@ func TestEditResourceNotfound(t *testing.T) {
 			Provider:     "google",
 			ClientSecret: "secret-2",
 			Mapper:       "file:///etc/config/kratos/google_schema.jsonnet",
-			Scope:        []string{"email", "profile"},
+			Scope:        []string{"email", "promockConfigMapV1.EXPECT().Update(gomock.Any(), cm, gomock.Any()).Times(1).Return(cm, nil)file"},
```

Comment:
changed on accident?

Reply:
fixed, it was on purpose to gauge reviewers' attention